### ISSUE TITLE
Only available for administrator not normal user.

### ISSUE THREAD

**Issue by nLz00** [JimDude7404/WSA-Windows-10#7]
I've installed as admin and can use it as logged in admin. i can install it as not-admin (with some errors) but can't use it as not-admin but as admin again.
How to install and use this as a not-admin?

**Comment by JimDude7404** [comment]
I do not think that is currently possible, as it is an unsigned package requiring developer mode. Anyone else here correct me if I am wrong. ps. I don't know everything.

**Comment by Subroutine7901** [comment]
Same issue, thanks for the info!

**Comment by JimDude7404** [comment]
You need to use it as an admin, sadly its not possible to use it without
being one.
